src/libraries/tests.proj

```diff
@@ -175,6 +175,9 @@
     <ProjectReference Include="..\mono\netcore\sample\iOS\Program.csproj"
                       BuildInParallel="false"
                       Condition="'$(ArchiveTests)' == 'true' and ('$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS')" />
+    <ProjectReference Include="..\mono\netcore\sample\wasm\console\WasmSample.csproj"
```

Comment:
missed this earlier, but @MaximLipnin can you please change this and the above to use

```suggestion
    <ProjectReference Include="$(MonoProjectRoot)netcore\sample\wasm\console\WasmSample.csproj"
```

Reply:
Yeah, thanks, I wanted to fix it during all the previous PRs but missed as well.

Reply:
Updated in https://github.com/dotnet/runtime/pull/46129